pkg/kubectl/cmd/util/helpers.go

```diff
@@ -203,26 +205,29 @@ func StandardErrorMessage(err error) (string, bool) {
 	return "", false
 }
 
-// MultilineError returns a string representing an error that splits sub errors into their own
-// lines. The returned string will end with a newline.
-func MultilineError(prefix string, err error) string {
-	if agg, ok := err.(utilerrors.Aggregate); ok {
+// MultilineErrorString returns a string representing an error that splits sub errors into their own
```

Comment:
There was a case (one out of two) where it was called with a nil err, giving strange output. I have reverted the change here and added a clause to avoid the nil on the callers side.